index.html

```diff
@@ -60,8 +63,8 @@ <h2>Provision and manage cloud infrastructure and services using kubectl</h2>
           alt="Stacks" id="stacksImg" />
         <h2>Created to power a more open cloud</h2>
         <p>
-          We built Crossplane to help organizations build their clouds like the cloud vendors build theirs—with a control plane. Crossplane is a CNCF sandbox project which extends the Kubernetes 
-          API to manage and compose infrastructure. Operators can encapsulate policies, permissions, and other guardrails behind a custom API line generated by Crossplane while application developers 
+          We built Crossplane to help organizations build their clouds like the cloud vendors build theirs—with a control plane. Crossplane is a CNCF sandbox project which extends the Kubernetes
```

Comment:
nitpick:
```suggestion
          We built Crossplane to help organizations build their clouds like the cloud vendors build theirs — with a control plane. Crossplane is a CNCF sandbox project which extends the Kubernetes
```

Reply:
Most style guides push for not using spaces around em dashes. I'm not that picky, though 🤷